src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -945,6 +952,7 @@ PeerImp::onWriteMessage(error_code ec, std::size_t bytes_transferred)
     if (!send_queue_.empty())
     {
         // Timeout on writes only
+        assert(send_queue_.front());
```

Comment:
Don't need this if the dummy message is not going to be used.

Reply:
Removed the unnecessary asserts.